readme.md

```diff
@@ -1,139 +1,99 @@
-# RavenDB - An ACID NoSQL Document Database
-This repository contains source code for the [RavenDB](https://ravendb.net/) document database. With a RavenDB database you can set up a NoSQL data architecture or add a NoSQL layer to your current relational database.
+<h1 align="center"><img src="docs/Server guide banner.png" alt="Ory Keto - Open Source & Cloud Native Access Control Server">RavenDB - Modern ACID NoSQL Database Solution</h1>
 
-![RavenDB Studio](docs/readmeScreenshot.png)
 
-## Supported Platforms
-- Windows
-- Linux
-- Docker
-- MacOS
-- Raspberry Pi
+<h4 align="center">    
+    <a href="https://discord.com/invite/ravendb">Discord 🌐💬🎙️</a> |
+    <a href="https://github.com/ravendb/ravendb/discussions">Discusssions 🎫</a> |
+    <a href="https://www.youtube.com/@ravendb_net">YouTube ▶️</a> 
+    <br/><br/>
+    <a href="live-test.ravendb.net/">🔌 Try from your browser, now! 🔌</a>
+    <br/><br/>
+    <a href="https://ravendb.net/docs">🎒👶 Documentation</a> |
+    <a htef="https://ravendb.net/articles/">📰 Articles</a> | 
+    <a htef="https://ravendb.net/book/"> Book 📖🍵 </a>
+    <br/><br/>
+    <a href="https://ravendb.net/buy#developer">💹💰 Free developer license </a>
```

Comment:
We could link right away to actual form at https://ravendb.net/license/request/dev
One step less